react-design/my-app/src/RotatingBanner.tsx

```diff
@@ -0,0 +1,56 @@
+type BannerProps = {
+  items: string[];
+  index: number;
+};
+
+export function BannerDisplay({ items, index }: BannerProps) {
+  return <h1>{items[index]}</h1>;
+}
+type SelectionsProps = {
+  num: number;
+  index: number;
+  onCustomClick: () => void;
+};
+export function BannerSelections({
+  num,
+  index,
+  onCustomClick,
+}: SelectionsProps) {
+  if (index === num) {
+    return (
+      <button onClick={() => onCustomClick()} className="c1-6 w">
+        {num}
+      </button>
+    );
+  } else {
+    return (
```

Comment:
The only difference between these two is the `w` in the className. It would be good to remove all this code duplication by dynamically computing the className.